.changeset/clean-plants-kick.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
One changeset is enough (you can remove these files). Also feel free to add a minor changeset with:
```
Add example that will fail
```

Reply:
Will do.